solutions/cses-1161.mdx

```diff
@@ -55,4 +55,39 @@ int main(){
 
 </CPPSection>
 
+<JavaSection>
+
+In Java, this can be done with a [PriorityQueue](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/PriorityQueue.html) as well. 
```

Comment:
not necessary?

Reply:
well in the c++ section this exists. If you switch to java then it will not be mentioned. Should I move the priority queue comment out of the language section or just remove this from java section?

Reply:
uh yeah
priority queues exist in all three languages
just put "we can do this w/ a priority queue" or smth